messaginginapp/src/main/java/io/customer/messaginginapp/di/DIGraphMessagingInApp.kt

```diff
@@ -13,7 +14,10 @@ internal val SDKComponent.gistApiProvider: GistApi
 internal val SDKComponent.gistProvider: InAppMessagesProvider
     get() = newInstance<InAppMessagesProvider> { GistInAppMessagesProvider(gistApiProvider) }
 
-fun SDKComponent.inAppMessaging(): ModuleMessagingInApp {
-    return modules[ModuleMessagingInApp.moduleName] as? ModuleMessagingInApp
+// We need to add extension functions to the CustomerIO class as this is how
+// customers will interact with in-app messaging module.
+@Suppress("UnusedReceiverParameter")
+fun CustomerIO.inAppMessaging(): ModuleMessagingInApp {
+    return SDKComponent.modules[ModuleMessagingInApp.MODULE_NAME] as? ModuleMessagingInApp
```

Comment:
The comment above should help explain reason for reverting it back to CustomerIO extension

Reply:
Reverted change as I realized we'll be removing `CustomerIO` dependency from in-app where the code will create problem. We need to improve how customers interact with these modules, but that's a different problem and we'll solve in another PRs.